.gitignore

```diff
@@ -47,4 +47,6 @@ _deps
 
 ### CMake Patch ###
 # External projects
-*-prefix/
\ No newline at end of file
+*-prefix/
+#binary
+nav
```

Comment:
This should not be required. One should build CMake projects outside the tree, i.e. by running 
```
mkdir build
cd build
cmake ..
```
You should exclude `build/` instead. This is good practice as (as far as I am aware) cmake is not very good at cleaning up after itself, especially when things go wrong... `rm -rf build` to the rescue.